http.kafka.cqrs/src/main/java/io/aklivity/example/cqrs/processor/FailedCommandProcessor.java

```diff
@@ -0,0 +1,46 @@
+package io.aklivity.example.cqrs.processor;
+
+import org.apache.kafka.common.header.Headers;
+import org.apache.kafka.common.header.internals.RecordHeaders;
+import org.apache.kafka.streams.processor.api.Processor;
+import org.apache.kafka.streams.processor.api.ProcessorContext;
+import org.apache.kafka.streams.processor.api.ProcessorSupplier;
+import org.apache.kafka.streams.processor.api.Record;
+
+import io.aklivity.example.cqrs.model.Command;
+
+public class FailedCommandProcessor implements ProcessorSupplier<String, Command, String, Object>
```

Comment:
Rename to `RejectCommandProcessorSupplier`, and move anonymous inner class to named inner class called `RejectCommandProcessor`.